packages/vuepress/vuepress-plugin-apidocs/index.js

```diff
@@ -182,24 +185,13 @@ export default ${JSON.stringify(typeLinks)}\n\n`.trim()
     },
 
     /**
-     * Split metadata per type and generate a JSON file for each one that gets
-     * loaded by Vuex on subsequent page loads once Vue takes over on the client.
+     * Copy metadata files previously generated in `ready` hook to output path
+     * so they can be loaded by Vuex on subsequent page loads once Vue takes
+     * over on the client.
      */
     async generated () {
       // @todo check context.markdown.$data.typeLinks for existence
-
-      const tempMetadataPath = path.resolve(context.tempPath, 'metadata')
-      fs.ensureDirSync(tempMetadataPath)
-      for (const version in processed) {
-        fs.ensureDirSync(path.join(tempMetadataPath, version))
-        for (const typeName in processed[version]) {
-          const metadata = metadataService.findMetadata(typeName, version)
-          const destPath = path.join(tempMetadataPath, version, `${typeName.toLowerCase()}.json`)
-          fs.writeFileSync(destPath, JSON.stringify(metadata))
-        }
-      }
-
-      await fs.copy(tempMetadataPath, path.resolve(context.outDir, 'metadata'))
+      await fs.copy(metadataDir, path.resolve(context.outDir, 'metadata'))
```

Comment:
instead of await, the parent method is async, so just return:

```suggestion
      return fs.copy(metadataDir, path.resolve(context.outDir, 'metadata'))
```